roles/openshift_metrics/templates/metrics-server-deployment.j2

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: "{{ openshift_metrics_server_project }}"
+  labels:
+    k8s-app: metrics-server
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+    version: "{{openshift_metrics_server_version}}"
+    metrics-infra: metrics-server
+spec:
+  selector:
+    matchLabels:
+      k8s-app: metrics-server
+      version: "{{openshift_metrics_server_version}}"
+  template:
+    metadata:
+      name: metrics-server
+      labels:
+        k8s-app: metrics-server
+        version: "{{openshift_metrics_server_version}}"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      priorityClassName: system-cluster-critical
+      serviceAccountName: metrics-server
+      containers:
+      - name: metrics-server
+        image: "k8s.gcr.io/metrics-server-amd64:{{openshift_metrics_server_version}}"
```

Comment:
nope.  Need to use our copy: https://github.com/openshift/aos-cd-jobs/pull/960

Reply:
Should `image` be a parameter in its entirety, dispensing with `..._version`?

Reply:
hmm... maybe